mobilecoind/src/utxo_store.rs

```diff
@@ -82,6 +82,8 @@ impl From<&KeyImage> for UtxoId {
 /// The outputs database.
 #[derive(Clone)]
 pub struct UtxoStore {
+    #[allow(dead_code)] // FIXME: Why does the compiler think this is unused?
+    /// LMDB Environment.
```

Comment:
I believe this is because env is never explicitly called over the course of the UtxoStore. (I believe) it's necessary for the struct to contain env because lmbd requires the databases to operate within the lifetime of the environment, thus it's never used explicitly outside of tests except to maintain the lifetime of the env. There should be some way to annotate this.